src/components/SearchReps.vue

```diff
@@ -16,71 +16,84 @@
+                <v-btn
+                  rounded
+                  dark
+                  :class="{ active: isActive }"
+                  :style="{
+                    backgroundColor:
+                      currentFilter === 'state' && isActive ? 'blue' : 'gray'
+                  }"
+                  v-on:click="FilterList('state')"
+                >
+                  State
+                </v-btn>
+              </v-col>
 
-              <v-btn
-                rounded
-                dark
-                :class="{ active: isActive }"
-                :style="{
-                  backgroundColor:
-                    currentFilter === 'county' && isActive ? 'blue' : 'gray'
-                }"
-                v-on:click="FilterList('county')"
-              >
-                County
-              </v-btn>
+              <v-col>
+                <v-btn
+                  rounded
+                  dark
+                  :class="{ active: isActive }"
+                  :style="{
+                    backgroundColor:
+                      currentFilter === 'county' && isActive ? 'blue' : 'gray'
+                  }"
+                  v-on:click="FilterList('county')"
+                >
+                  County
+                </v-btn>
+              </v-col>
 
-              <v-btn
-                rounded
-                dark
-                class="ui button toggle"
-                :style="{
-                  backgroundColor:
-                    currentFilter === 'municipality' && isActive
-                      ? 'blue'
-                      : 'gray'
-                }"
-                v-on:click="FilterList('municipality')"
-              >
-                Local
-              </v-btn>
+              <v-col>
+                <v-btn
+                  rounded
+                  dark
+                  class="ui button toggle"
+                  :style="{
+                    backgroundColor:
+                      currentFilter === 'municipality' && isActive
+                        ? 'blue'
+                        : 'gray'
+                  }"
+                  v-on:click="FilterList('municipality')"
+                >
+                  Local
+                </v-btn>
+              </v-col>
             </v-row>
 
-            <v-btn
-              :to="{
-                name: 'Reps',
-                params: { postalCode: postalCode }
-              }"
-              v-on:click="CreateRepList()"
-              clickclass="mr-4"
-              >Submit
-            </v-btn>
+            <v-row>
+              <v-col>
+                <v-btn
+                  :to="{
+                    name: 'Reps',
+                    params: { postalCode: postalCode }
+                  }"
+                  v-on:click="CreateRepList()"
+                  clickclass="mr-4"
+                  >Submit
+                </v-btn>
+              </v-col>
+            </v-row>
           </v-card-text>
         </v-card>
+
```

Comment:
Here, if we could put all the buttons (with proper margins amongst themselves) within the same column, we wouldn't see a lot of gap between them as we see here:

![image](https://user-images.githubusercontent.com/30315706/196333507-329ad74a-8364-42c2-abf9-19570e3c6e6b.png)

As the window size keeps increasing, the gap between the filter buttons keeps increasing too. It would be better if the gap remains consistent and if all the buttons are stored within a single column contained in the row component.